cpp/include/cudf/scalar/scalar_factories.hpp

```diff
@@ -78,7 +78,7 @@ std::unique_ptr<scalar> make_timestamp_scalar(
  */
 std::unique_ptr<scalar> make_duration_scalar(
   data_type type,
-  rmm::cuda_stream_view stream      = cudf::get_default_stream(),
```

Comment:
There are few places where `cudf::column` is used instead of `column`. At someplaces like this, `cudf:` is removed. What's the convention followed here?

Reply:
I am not sure why the namespacing changes happened. Let me do another pass where I fully qualify all the namespaces, then see if I can get clang-tidy to remove them and tell me why. Part of me suspects that this might have happened during some iterative runs of the autofixer but that in the current state it won't care about them.

Reply:
OK, I tested this in https://github.com/rapidsai/cudf/pull/15894/commits/82b73a019f02b09f43c89f6045ca030db85ec2d3, and as I suspected the autofix did not change it again. Let's stick with fully qualified namespaces, I think we all generally prefer that.